AUTHORS

```diff
@@ -299,3 +299,4 @@ Yohan Jarosz <yohanjarosz@yahoo.fr>
 Yonas Habteab <yonas.habteab@icinga.com>
 Zachary McGibbon <zachary.mcgibbon@gmail.com>
 Zoltan Nagy <abesto@abesto.net>
+Maciej Dems <macdems@gmail.com>
```

Comment:
Seems like you're using a different email to push the changes that's the GitHub action is complaining about. `Maciej Dems <maciej.dems@p.lodz.pl>`

While you're fixing this, please respect the order of the authors' names (ordered alphabetically) and push these changes also as part of the second commit, not the first one. Thanks!